docs/docs/connectors/slack.mdx

```diff
@@ -12,7 +12,7 @@ Once you have them you can add these to your `credentials.yml`.
 
 **How to get the Slack credentials:** You need to set up a Slack app.
 
-1. To create the app go to [https://api.slack.com/apps](https://api.slack.com/apps) and click
+1. To create the app go to [Your Apps](https://api.slack.com/apps) and click
```

Comment:
From an a11y POV this link is a little bare, since the text "Your Apps" which would be read out-of-context by someone navigating with assistive tech, is not self-explanatory. You can add a title to a link like this
```md
To create the app go to [Your Apps](https://api.slack.com/apps "The Your Apps section of your Slack interface")
```

Reply:
Ooh i did not know this! that's super helpful